include/fiction/algorithms/physical_design/hexagonalization.hpp

```diff
@@ -865,7 +861,7 @@ class hexagonalization_impl
                     }
                     else
                     {
-                        throw hexagonalization_route_inputs_error("Extending POs to the bottom border failed.");
+                        throw hexagonalization_io_pin_routing_error("Extending POs to the bottom border failed.");
```

Comment:
Is it possible to give a little more information on what failed exactly and why?